docs/software/cw/netcdf-tools.md

```diff
@@ -0,0 +1,90 @@
+[](){#ref-uenv-netcdf-tools}
+# netcdf-tools
+
+The `netcdf-tools` uenv provides a set of CLI tools and GUI tools frequently used in climate and weather workflows.
+
+The release schedule is not fixed, with new releases when required.
+
+| version            | node types        | system                                  | status |
+|--------------------|-------------------|-----------------------------------------|--------|
+| 2025               | zen2, gh200       | daint, eiger, santis, clariden          | :white_check_mark: |
+| 2024               | zen2, gh200       | daint, eiger, santis, clariden          | **DEPRECATED** |
```

Comment:
Do we need to just remove this uenv if it's completely broken? Or can it still be used for _some_ things?